modules/oe_theme_helper/tests/src/Functional/NodeViewRoutesMetadataTest.php

```diff
@@ -68,18 +69,26 @@ public function setUp(): void {
       'view latest version',
       'view any unpublished content',
     ]);
+
+    FilterFormat::create([
+      'format' => 'full_html',
+      'name' => 'Full HTML',
+    ])->save();
   }
 
   /**
    * Tests that the correct metadata is retrieved on node view routes.
    */
   public function testNodeRoutes(): void {
     // Create a published revision for a node.
-    $published_revision_body = $this->randomString();
+    $published_revision_body = '<u>Custom page</u> introduction with <sub>rich</sub><sup>text</sup><mark>highlighted</mark>.';
```

Comment:
Why are we testing the exact same things as in `\Drupal\Tests\oe_theme_helper\Functional\PageHeaderBlockTest::testRendering`? Could you please just update the mentioned test method and remove all the changes in this class?

Reply:
I have found that test in `\Drupal\Tests\oe_theme_helper\Functional\PageHeaderBlockTest::testRendering` doest take in account allowed tags in `\Drupal\oe_theme_helper\Plugin\PageHeaderMetadata\NodeViewRoutesBase::getIntroductionMetadata`

Reply:
So it will always green even if we tried to put tags which should be stripped.

Reply:
You're right 😞